src/hotspot/share/gc/shenandoah/heuristics/shenandoahHeuristics.cpp

```diff
@@ -304,6 +436,155 @@ double ShenandoahHeuristics::time_since_last_gc() const {
 
 bool ShenandoahHeuristics::in_generation(ShenandoahHeapRegion* region) {
   return (_generation->generation_mode() == GLOBAL)
-      || (_generation->generation_mode() == YOUNG && region->affiliation() == YOUNG_GENERATION)
-      || (_generation->generation_mode() == OLD && region->affiliation() == OLD_GENERATION);
+    || (_generation->generation_mode() == YOUNG && region->affiliation() == YOUNG_GENERATION)
+    || (_generation->generation_mode() == OLD && region->affiliation() == OLD_GENERATION);
```

Comment:
I wouldn't change indentation here, or if it needs changing, then line up the || with the _generation above. Iow, shift it 4 spaces to the right instead of 2 spaces to the left.

Reply:
I think I've fixed this now.  Auto-indent of my editor got carried away and I apparently lost track of original spacing.